src/components/about/index.js

```diff
@@ -0,0 +1,3 @@
+import About from './About';
+
+export default About;
```

Comment:
Looks like we're missing a blank line at the end of this file—I'll just throw one in on the next frontend PR, so okay to ignore.